dev-python/ciso8601/Manifest

```diff
@@ -0,0 +1,3 @@
+DIST ciso8601-2.1.3.tar.gz 15726 BLAKE2B 2122c6b677ef0386bfab30fb76d746ff0d22e402464a3e0b928b71d0c22cf9dc26503dba52750015f45b4397ebd1dca16400873d2f67ce38dac2b5fbed2402dc SHA512 9342cf9b2ded1077a2c3906527ae4ac05dfc0e8104f1c85e041d636caa1180df390f01b2d87a33a4c197f4467a25bda1cd0136eb48585b127662cde7d0ffda7e
+EBUILD ciso8601-2.1.3.ebuild 671 BLAKE2B d3b7585c062217a25f1c68c9b7aeea07ce0099a850ddc35e159c6db29aa884efc2f68bae1b226b8546f7fc848dec52f9415f925326ae557b293a161d6d135208 SHA512 811bb77f6d682bb0bae96b09bbe59ccdd2275790bf5aa67493b67652dc79438893a74699d65e1e07da41199be0603531f6812ae2370fd2052eda860a36198396
+MISC metadata.xml 404 BLAKE2B 19edddec02504a5f05ac1a5ce6dba6faec8031c4a82d3c3ed380a478157213bb6bcd80ae9410c8f92bb2f76bd784d207ba41917471d4781f5f5c2ace48f169c0 SHA512 d3541c13ee1f7a79e3b096bb19d4a740792938de1752784387d6e6f8a31f166a216cdb60dbfc2cf68121c0c9979e5839a91a647411ce5fb831b93ead8afbe98f
```

Comment:
Your metadatas seems wrong... Have you run ebuild **packagename.ebuild** manifest

Reply:
This comment is for every metadata you have pushed

Reply:
You have to run ebuild only with .ebuild file, not with metadata.xml